HousingSearchApi/V1/Boundary/Responses/GetAllTenureListResponse.cs

```diff
@@ -0,0 +1,28 @@
+namespace HousingSearchApi.V1.Boundary.Responses
+{
+    public class GetAllTenureListResponse : GetTenureListResponse
+    {
+        private string _lastHitId;
+        private string _lastHitTenureStartDate;
+
+        public void SetLastHitId(string lastHitId)
+        {
+            _lastHitId = lastHitId;
+        }
+
+        public string LastHitId()
+        {
+            return _lastHitId;
+        }
```

Comment:
We want the search gateway to add the `LastHitId` and `LastHitTenureStartDate` to the `GetAllTenureListResponse` to be used later in the API response. However we don't want those properties to appear in the response inside `results` object, but at the top level.

The original get/set implementation followed the same pattern used elsewhere in response objects, but I've now updated it in the [#ea51f55](ea51f55a5f9302148f38d582c86e8f30b716da29) to ensure the properties only appear where we want while the implementation looks more like standard C# property handling. By making the properties internal in `GetAllTenureListResponse` they won't be included in the API response amongst the result automatically as it would be the case if they were public.

Reply:
Oh, I see.
In which case this can simplified a lot by simply adding the `[JsonIgnore]` [attributes](https://www.newtonsoft.com/json/help/html/PropertyJsonIgnore.htm) above the fields.

Reply:
I was debating whether to go down that route, but because that approach wasn't used elsewhere in the API I initially decided againts it. However I do agree that using `[JsonIgnore]` makes things easier to manage and also makes it more obvious that we are doing that for "presentational" reasons. This approach is used in other APIs as well, so no reason to avoid it here really. I'll leave the other objects using private properties as is to keep the scope of these changes limited to this end point, but that's something we can look into changing if we have to update those models at some point. I have made the properties public in [#51b7b57](https://github.com/LBHackney-IT/housing-search-api/pull/226/commits/51b7b571f736d43ae01001e848f2a64fdfc923f5). Also tweaked the related test to be more specific. I did realise too that setting up the private properties like that is not necessary and sticking with the auto-implemented properties i.e. `internal string LastHitId { get; set; }` would have worked just the same. Anyway I think this option is better and makes it more obvious as to why we are handling those properties that way.